src/app.js

```diff
@@ -189,7 +190,15 @@ var App = React.createClass({
       }.bind(this)
     );
   },
-
+  _onRun: function() {
```

Comment:
Check out TransformOutput, this is where the code should belong. See https://github.com/cpojer/esprima_ast_explorer/blob/gh-pages/src/TransformOutput.js#L37 – we should have the try/catch for the transform in there – if it fails, we can console.log the error + show a red bar on top of the transform output (similar to how we show an error when the other field fails to parse)